poucave/main.py

```diff
@@ -119,7 +120,8 @@ def run_check(conf):
     params = conf.get("params", {})
     func = getattr(importlib.import_module(module), "run")
     pool = concurrent.futures.ThreadPoolExecutor()
-    success, data = pool.submit(asyncio.run, func(None, **params)).result()
+    fakerequest = make_mocked_request(method="GET", path="")
+    success, data = pool.submit(asyncio.run, func(fakerequest, **params)).result()
```

Comment:
This was a bit confusing. Why does this have to be an actual request, since the path and method are constant? Maybe we should just extract the query parameters as a multidict or something, and pass that on?

Reply:
Yes, you're right, I wanted to leave every possibilities for a check (use requests headers, query params, etc.) but passing the request query params as dict should do it in most cases.